nekoyume/Assets/_Scripts/BlockChain/ActionManager.cs

```diff
@@ -426,6 +434,7 @@ public bool TryPopActionEnqueuedDateTime(Guid actionId, out DateTime enqueuedDat
                 .Where(eval => eval.Action.Id.Equals(action.Id))
                 .First()
                 .ObserveOnMainThread()
+                .DoOnCompleted(() => Analyzer.Instance.FinishTrace(sentryTrace))
```

Comment:
Is it OK to handle `sentryTrace` only when this task succeeds and completed and not the error situation?
`DoOnCompleted()` callback will not invoked when the stream will error.

The "Unity/Create CombinationConsumable" event is used regardless of an error. So I think it would be better not to use the return value(`sentryTrace`) and then the `FinishTrace()` method will invoked internaly.

Reply:
i didn't know how it works on error, then how about using `Finally()` instead of `DoOnCompleted()`?

Reply:
It's also good idea!